functests/1_performance/performance.go

```diff
@@ -225,6 +227,72 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 		})
 	})
 
+	Context("RPS configuration", func() {
+		It("[test_id:xxxxx][ Should have the correct RPS configuration", func() {
+			if profile.Spec.CPU != nil && profile.Spec.CPU.Reserved != nil {
```

Comment:
on general we prefer to bail out early if the function preconditions aren't met. Not a big deal though.

Reply:
I will change, thanks